examples/clients/abe/BUILD.bazel

```diff
@@ -1,25 +1,51 @@
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load(
+    "//examples/clients:clients.bzl",
+    "swagger_codegen",
+    "gen_file_integrity_test",
+)
 
 package(default_visibility = ["//visibility:public"])
 
+# keep
 go_library(
     name = "go_default_library",
     srcs = [
-        "a_bit_of_everything_nested.go",
-        "a_bit_of_everything_service_api.go",
-        "api_client.go",
-        "api_response.go",
-        "camel_case_service_name_api.go",
-        "configuration.go",
-        "echo_rpc_api.go",
-        "echo_service_api.go",
-        "examplepb_a_bit_of_everything.go",
-        "examplepb_body.go",
-        "examplepb_numeric_enum.go",
-        "nested_deep_enum.go",
-        "protobuf_empty.go",
-        "sub_string_message.go",
+        ":gen_client",
     ],
     importpath = "github.com/grpc-ecosystem/grpc-gateway/examples/clients/abe",
     deps = ["@com_github_go_resty_resty//:go_default_library"],
 )
+
+SRCS = [
```

Comment:
Could this be done as a glob? If we do it as a glob and use he comment I left in the generator, then if something changes in the swagger definition and creates a new `.go` file it'll be updated automatically